playground/infrastructure/helper.py

```diff
@@ -98,11 +99,17 @@ def find_examples(root_dir: str, subdirs: List[str], sdk: SdkEnum) -> List[Examp
             for filename in files:
                 filepath = os.path.join(root, filename)
                 try:
-                    example = _load_example(
-                        filename=filename, filepath=filepath, sdk=sdk
-                    )
-                    if example is not None:
-                        examples.append(example)
+                    try:
+                        example = _load_example(
+                            filename=filename, filepath=filepath, sdk=sdk
+                        )
+                        if example is not None:
+                            examples.append(example)
+                    except pydantic.ValidationError as err:
+                        if len(err.errors()) > 1:
+                            raise
+                        if err.errors()[0]["msg"] == "multifile is True but no files defined":
+                            logging.warning("incomplete multifile example ignored %s", filepath)
```

Comment:
Should we raise an exception in the else clause here? Are there other cases where we want to ignore an error?

Reply:
good point, fixed!